InferenceSystem/deploy/point-robinson.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
```

Comment:
Are the new containers already deployed to the cluster using this yaml file? If not, please feel free to do so.

Reply:
They are deployed and are streaming data! :)